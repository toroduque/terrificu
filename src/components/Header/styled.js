import styled from 'styled-components';

export const HeaderWrapper = styled.div`
    display: flex;
    align-items: center;
    justify-content: space-between;
    padding: 0 1.6rem;
    height: 5.3rem;
    box-shadow: 0 1.2rem 2.4rem rgba(0,0,0,0.12);
`
