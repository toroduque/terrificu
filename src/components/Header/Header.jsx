/* eslint-disable */

import React, { Component } from "react";
import Image from "../Image";
import Menu from "./MenuIcon";
import * as styled from "./styled";

class Header extends Component {
    state = {
        isMenuOpen: false
    }

    setIsMenuOpen() {
        this.setState({ isMenuOpen: !this.state.isMenuOpen })
    }

    render() {
        return (
            <styled.HeaderWrapper>
                <Image />
                <Menu
                    onClick={() => this.setIsMenuOpen()}
                    isMenuOpen={this.state.isMenuOpen}
                />
            </styled.HeaderWrapper>
        );
    }
}

export default Header;
