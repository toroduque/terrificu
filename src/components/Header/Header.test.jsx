import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header'

describe('Header component', () => {
    it('Should match the snapshot', () => {
        const component = shallow(<Header />)

        expect(component).toMatchSnapshot()
    })
})
