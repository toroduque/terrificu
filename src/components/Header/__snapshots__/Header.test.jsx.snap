// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Should match the snapshot 1`] = `
<div>
  Hola mundo!
</div>
`;
